CHANGELOG.md

```diff
@@ -386,6 +404,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Fixed `self.optimizers()` not returning a single optimizer if it had been wrapped ([#8326](https://github.com/PyTorchLightning/pytorch-lightning/pull/8326))
 
 
+- Fixed the `on_after_backward` hook not getting called when using manual optimization and no plugins ([#8328](https://github.com/PyTorchLightning/pytorch-lightning/pull/8328))
+
+
+- Fixed the `LightningModule.backward` hook only getting called with the `apex` plugin ([#8328](https://github.com/PyTorchLightning/pytorch-lightning/pull/8328))
```

Comment:
I don't understand, where else would the backward happen?
Can you point me to the changes?

Reply:
Missed this:

```suggestion
- Fixed the `LightningModule.backward` hook only getting called with the `apex` plugin when using manual optimization ([#8328](https://github.com/PyTorchLightning/pytorch-lightning/pull/8328))
```

Apex was the only one calling the `LightningModule` hook:

https://github.com/PyTorchLightning/pytorch-lightning/blob/e9d0fe867f7aec14689d7069b3fa79ef047448e9/pytorch_lightning/plugins/precision/apex_amp.py#L78-L79

As it is the only plugin to override `backward` from the base class.

The base class does:

https://github.com/PyTorchLightning/pytorch-lightning/blob/e9d0fe867f7aec14689d7069b3fa79ef047448e9/pytorch_lightning/plugins/precision/precision_plugin.py#L76-L80

Which as you can see, only calls it if `automatic_optimization`